app/javascript/components/AssignmentButton/index.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { apiPost, apiPatch, apiDelete } from 'utils/axios';
+import { Button } from '@material-ui/core/';
+
+class AssignmentButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleCreate() {
+    const payload = {
+      title: this.state.title,
+      description: this.state.description,
+      assigned_to_ids: [10, 22, 14, 12],
+    };
+    apiPost(`/api/assignments/`, { assignment: payload })
+      .then(resp => console.log(resp))
+      .catch(e => console.log(e));
+  }
+
+  handleUpdate() {
+    const payload = { title: 'NEW TITLE WOOHOOOOO' };
+    apiPatch(`/api/assignments/${5}`, { assignment: payload })
+      .then(resp => console.log(resp))
+      .catch(e => console.log(e));
+  }
+
+  handleDeleteSubmit() {
+    apiDelete(`/api/assignments/${3}`)
+      .then(resp => console.log(resp))
+      .catch(e => console.log(e));
+  }
+
+  handleTemplateCreate() {
+    const payload = { title: 'SECOND NEW TITLE!', description: 'PLEASE WORK!' };
+    apiPost('/api/assignments/templates', { assignment: payload })
+      .then(resp => console.log(resp))
+      .catch(e => console.log(e));
+  }
+
+  handleTemplateUpdate() {
+    const payload = { title: 'THIS IS A CHANGED TITLE TWO!' };
+    apiPatch(`api/assignments/templates/1`, { assignment: payload })
+      .then(resp => console.log(resp))
+      .catch(e => console.log(e));
+  }
+
+  handleTemplateDelete() {
+    apiDelete(`api/assignments/templates/1`)
```

Comment:
hardcoded on purpose?

Reply:
yeah these were hardcoded on purpose. They're supposed to be deleted before merging this was just to make it easier to test functionality for you guys